index.html

```diff
@@ -9,11 +9,13 @@
     <input type="text" id="dpi" placeholder="DPI (default=2400)" /><br/>
     <input type="text" id="margin" placeholder="Margin (default=20)" /><br/>
     <input type="text" id="turns" placeholder="Turns (default=3)" /><br/>
-    <input type="text" id="ant_length" placeholder="Antenna length (default=10)" /><br/>
+    <input type="text" id="ant_height" placeholder="Antenna height (default=10)" /><br/>
```

Comment:
heh, i wanted to use `height` but somehow people were referring to `length` in documents, that's why I used `length`. But `height` is more clear so let's keep it :)